spec/services/healthcheck/pvb_api_check_spec.rb

```diff
@@ -24,15 +24,16 @@
     before do
       allow_any_instance_of(PrisonVisits::Api).
```

Comment:
If we could get rid of `allow_any_instance_of(PrisonVisits::Api)` for `allow(PrisonVisits.instance)` that would be great. As a general rule never use `allow_any_instance_of` because you might not be stubbing what you expect